styles/omni-ruler.less

```diff
@@ -1,6 +1,6 @@
 @import "syntax-variables";
 
-atom-text-editor::shadow {
+atom-text-editor {
```

Comment:
The Atom deprecation warning referenced in #14 mentioned using this selector: `atom-text-editor.editor .omni-ruler`.

Should this line should be `atom-text-editor.editor`?